src/hooks/useIotInfo.ts

```diff
@@ -21,11 +20,8 @@ export const useIotInfo = (
 ): UseAccountState<IotHotspotInfoV0> | undefined => {
```

Comment:
Not the right return type. I'd just omit the return type entirely and let typescript figure it out